apps/src/lib/ui/accounts/ChangeUserTypeForm.story.jsx

```diff
@@ -4,59 +4,50 @@ import {action} from '@storybook/addon-actions';
 
 const DEFAULT_PROPS = {
   values: {
-    currentEmail: '',
+    email: '',
```

Comment:
The way this existed before was broken, this key should be `email` instead of `currentEmail`.

Reply:
Wow. I wonder why it worked before. Perhaps this API change suddenly paid for itself.